### PR TITLE
Replace configuration variables with configuration

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Part of renamethon. Looking for suggestions on how this feels, personally it feels weird but that might be bias of seeing config variables all over the place. 

Relevant conversation: https://www.flowdock.com/app/rulemotion/i-docs/threads/qWyLIT5_yKbnwPJOYROmhbq8lxi

TODO

- [x] update the screenshots
- [x] Add an additional heading for custom variables for the dashboard
    - [x] Also, do a side quest for Matthew and create an anchor link for documenting custom variables. 
- [x] If and when Chris approves, need to make changes onto the Dashboard to click correct screenshots (https://github.com/balena-io/balena-ui/pull/5246)

### CODE REVIEW

File: pages/learn/manage/configuration.md [hunk 1]
```diff
@@ -24,53 +24,53 @@ BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }}
...
-In addition to these values, there may be some device-type specific configuration variables that can be set. For example, these are a few of the values that apply to Raspberry Pi devices, corresponding to the contents of the [Raspberry Pi `config.txt` file](https://www.raspberrypi.org/documentation/configuration/config-txt/README.md):
+In addition to these values, there may be some device-type specific configuration that can be set. For example, these are a few of the values that apply to Raspberry Pi devices, corresponding to the contents of the [Raspberry Pi `config.txt` file](https://www.raspberrypi.org/documentation/configuration/config-txt/README.md):
 
 {{> "general/config-variables-pi" }}
 
-You can find more information on updating `config.txt` through configuration variables in our [Advanced Boot Configuration Guide][boot-config-guide].
+You can find more information on updating `config.txt` through the configuration tab in our [Advanced Boot Configuration Guide][boot-config-guide].
 
-## Managing fleet configuration variables
+## Managing fleet level configuration
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:33]:
```suggestion
## Fleet configuration management
```

File: pages/learn/manage/configuration.md [hunk 2]
```diff
@@ -24,53 +24,53 @@ BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }}
...
 
-Configuration variables defined at the fleet level control the behavior of any devices running in that fleet unless they are redefined with a device configuration variable of the same name.
+Variables defined on the fleet wide configuration control the behavior of any devices running in that fleet unless they are redefined with a device level variable of the same name.
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:35]:
```suggestion
Configuration defined at the fleet level controls the behavior of any devices running in that fleet unless it is overridden with device configuration of the same name.
```

File: pages/learn/manage/configuration.md [hunk 3]
```diff
@@ -24,53 +24,53 @@ BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }}
...
 
-If you want to change the default value for one of the prepopulated configuration variables, make sure you are in the *Fleet Configuration* tab, then click *activate* for the variable you wish to define:
+If you want to change the default value for one of the prepopulated configuration, make sure you are in the *Configuration* tab on the fleet level, then click *activate* for the variable you wish to define:
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:37]:
```suggestion
If you want to change the default value for any of the pre-populated configuration, make sure you are in the *Configuration* tab on the fleet, then click *activate* for the variable you wish to define:
```

File: pages/learn/manage/configuration.md [hunk 4]
```diff
@@ -24,53 +24,53 @@ BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }}
...
+<img alt="Add custom fleet configuration" src="/img/configuration/variable_editor_config.png" width="60%">
 
-If you have already defined configuration variables at the device level, they will appear in a list below your fleet configuration variables. You can easily apply a device value to the entire fleet by clicking *Define fleet-wide*:
+If you have already defined configuration at the device level, they will appear in a list below your fleet wide configuration. You can easily apply a device value to the entire fleet by clicking *Define fleet-wide*:
 
-<img alt="Define device configuration variables fleet-wide" src="/img/configuration/define_app_wide.png" width="100%">
+<img alt="Define device level configuration fleet-wide" src="/img/configuration/define_app_wide.png" width="100%">
 
-## Managing device configuration variables
+## Managing device level configuration
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:55]:
```suggestion
## Device configuration management
```

File: pages/learn/manage/configuration.md [hunk 5]
```diff
@@ -24,53 +24,53 @@ BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }}
...
 
-Device configuration variables define the behavior of a single device. If both the fleet and the device have a configuration variable of the same name, the code on the device will use the value of the device configuration variable. In other words, device configuration variables redefine (or override) fleet configuration variables.
+Device specific configuration define the behavior of a single device. If both the fleet and the device have configuration of the same name, the code on the device will use the value of the device level configuration. In other words, device level configuration variables redefine (or override) fleet wide configuration.
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:57]:
```suggestion
Device configuration defines the behavior of a single device. If both the fleet and the device have configuration of the same name, the code on the device will use the value given in the device configuration. In other words, device level configuration redefines (or overrides) fleet wide configuration.
```

File: pages/reference/OS/advanced.md [hunk 6]
```diff
@@ -36,17 +36,17 @@ Variables that start with the `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` pref
...
+To manage the configuration via the dashboard, add/edit variables via the *Configuration* tab on the [fleet][config-vars-fleet] or [device][config-vars-device] level. The variables with the same named defined at the device level, will override the variables defined fleet wide. See [configuration documentation][config-vars] for more details.
 
 Configuration values that are prepopulated may be edited or toggled (enabled/disabled):
 
-<img alt="Prepopulated device configuration variables" src="/img/configuration/default_configuration.png" width="100%">
+<img alt="Prepopulated device configuration" src="/img/configuration/default_configuration.png" width="100%">
 
-Custom configuration variables may be added by clicking _Add custom variable_ and entering a name with the prefix of `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` as shown below:
+Custom configuration may be added by clicking _Add custom variable_ and entering a variale with name having the prefix of `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` as shown below:
```
Comment (chrisys) [inline pages/reference/OS/advanced.md:45]:
```suggestion
Custom configuration may be added by clicking _Add custom variable_ and entering a variable with name having the prefix of `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` as shown below:
```

File: pages/learn/deploy/deploy-with-balena-button.md [hunk 7]
```diff
@@ -83,9 +83,11 @@ version: 3.5.2
     - `data`:
       - `url` - The URL of the asset that is being uploaded.
 - `data`
-  - `applicationEnvironmentVariables` - Variables allow you to provide runtime configuration without having to modify your source code.
-  - `applicationConfigVariables` - Configuration variables allow you to provide runtime configuration to the host OS and supervisor. These variables all begin with `BALENA_` or `RESIN_`.
+  - `applicationEnvironmentVariables` - [Variables][variable] allow you to provide runtime configuration without having to modify your source code.
```
Comment (chrisys) [inline pages/learn/deploy/deploy-with-balena-button.md:86]:
```suggestion
  - `applicationEnvironmentVariables` - [Variables][variables] allow you to provide runtime configuration without having to modify your source code.
```

File: pages/learn/manage/configuration.md [hunk 8]
```diff
@@ -28,52 +28,59 @@ In addition to these values, there may be some device-type specific configuratio
...
+<img alt="Edit device configuration" src="/img/configuration/edit_device_config.png" width="60%">
 
-The device configuration variable list includes the prepopulated default values. Variables that can be enabled or disabled can be easily switched with the provided toggles. Other values can be edited by clicking the small edit (pencil) icon.
+To remove the device level configuration, and reset it to its default value, click the delete (trash can) icon.
 
-<img alt="Device configuration variables" src="/img/configuration/device_config_variables.png" width="100%">
+### Adding custom configuration
 
-Clicking the edit icon will pop up a small dialog for editing the value:
+Custom configuration section can be used to modify configuration options beyond the ones pre-populated for your device using the balenaCloud dashboard. Examples include, [modifying config.txt using configuration variables][boot-config-guide] for Raspberry Pi devices.
```
Comment (chrisys) [inline pages/learn/manage/configuration.md:63]:
```suggestion
The custom configuration section can be used to modify configuration options beyond the ones pre-populated for your device using the balenaCloud dashboard. Examples include [modifying config.txt using configuration variables][boot-config-guide] for Raspberry Pi devices.
```

File: pages/learn/deploy/deploy-with-balena-button.md (hunk 7 above)
Comment (vipulgupta2048) [reply pages/learn/deploy/deploy-with-balena-button.md:86]: Good catch

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**